test/e2e_validate_data/test_validate_data.py

```diff
@@ -0,0 +1,176 @@
+from functools import lru_cache
+from pathlib import Path
+from typing import Iterator
+
+import pytest
+
+from dsp_tools.cli.args import ServerCredentials
+from dsp_tools.commands.project.create.project_create import create_project
+from dsp_tools.commands.validate_data.models.input_problems import ContentRegexViolation
+from dsp_tools.commands.validate_data.models.input_problems import MaxCardinalityViolation
+from dsp_tools.commands.validate_data.models.input_problems import MinCardinalityViolation
+from dsp_tools.commands.validate_data.models.input_problems import NonExistentCardinalityViolation
+from dsp_tools.commands.validate_data.models.input_problems import ValueTypeViolation
+from dsp_tools.commands.validate_data.models.validation import ValidationReports
+from dsp_tools.commands.validate_data.reformat_validaton_result import reformat_validation_graph
+from dsp_tools.commands.validate_data.validate_data import _get_validation_result
+from test.e2e_validate_data.setup_testcontainers import get_containers
+
+CREDS = ServerCredentials("root@example.com", "test", "http://0.0.0.0:3333")
+LOCAL_API = "http://0.0.0.0:3333"
+SAVE_GRAPHS = False
```

Comment:
*idea* How about renaming this variable to `DONT_SAFE_GRAPHS`?

I assume the name of the function parameter is something like save_graph. However, when using this variable during the function call it is misleading as the name `SAVE_GRAPH` suggests the value to be true.